Sources/IO/Misc/PDBReader/example/index.js

```diff
@@ -26,6 +26,7 @@ const sphereActor = vtkActor.newInstance();
 const stickActor = vtkActor.newInstance();
 
 filter.setInputConnection(reader.getOutputPort());
+filter.setHideElement('H');
```

Comment:
Good call indeed! I'm going to work on the version that you advise providing multiple string separated by a coma: `filter.setHideElements('H', 'O');` which makes more sense indeed